src/web/central_command/client/components/AXUI.jsx

```diff
@@ -315,31 +324,42 @@ export default class AXUI extends React.Component {
 
 		const { baseLayerCollection } = this.state;
 
-		[
-			new OlTileLayer({
-				title: 'NOAA Charts',
-				type: 'base',
-				source: new OlTileWMS({
-					url: 'https://gis.charttools.noaa.gov/arcgis/rest/services/MCS/ENCOnline/MapServer/exts/MaritimeChartService/WMSServer',
-					// params: {'LAYERS': 'topp:states', 'TILED': true},
-					serverType: 'geoserver',
-					transition: 0,
-				}),
-			}),
-			new OlTileLayer({
-				title: 'Google Satellite & Roads',
-				type: 'base',
-				source: new OlSourceXYZ({ url: 'http://mt1.google.com/vt/lyrs=y&x={x}&y={y}&z={z}' }),
-			}),
-			new OlTileLayer({
-				title: 'OpenStreetMap',
-				type: 'base',
-				source: new OlSourceOsm()
-			})
-		].forEach((layer) => {
-			makeLayerSavable(layer);
-			baseLayerCollection.push(layer);
-		});
+                [
+                        new OlTileLayer({
+                                title: 'Google Satellite & Roads',
+                                //type: 'base',
+                                opacity: 0.5,
+                                zIndex: 9,
+                                source: new OlSourceXYZ({ url: 'http://mt1.google.com/vt/lyrs=y&x={x}&y={y}&z={z}' }),
+                        }),
+                        new OlTileLayer({
+                                title: 'OpenStreetMap',
+                                //type: 'base',
+                                zIndex: 9,
+                                opacity: 0.3,
+                                source: new OlSourceOsm()
+                        }),
+                        new OlTileLayer({
+                                title: 'NOAA ENC Charts',
```

Comment:
I tried turning this layer on but the layer did not show

Reply:
Strange...  it loads for me!

Reply:
Hmm let me try again

Reply:
Works for me as well! Does the open street maps layer looks darker now for you Ed?